src/Illuminate/Database/Connection.php

```diff
@@ -821,16 +821,17 @@ protected function isUniqueConstraintError(Exception $exception)
      * @param  string  $query
      * @param  array  $bindings
      * @param  float|null  $time
+     * @param  int|null  $count
      * @return void
      */
-    public function logQuery($query, $bindings, $time = null)
+    public function logQuery($query, $bindings, $time = null, $count = null)
```

Comment:
Technically the change to the signature here is breaking but given that it's very unlikely someone if overwriting this we might be able to do this in 10.x